torch/onnx/symbolic_helper.py

```diff
@@ -23,27 +24,11 @@
 from torch import _C
 
 # Monkey-patch graph manipulation methods on Graph, used for the ONNX symbolics
-from torch.onnx import _constants, _type_utils, errors
+from torch.onnx import _constants, _type_utils, errors, utils
 from torch.onnx._globals import GLOBALS
 from torch.onnx._internal import _beartype, jit_utils
 from torch.types import Number
 
-__all__ = [
-    "args_have_same_dtype",
-    "cast_pytorch_to_onnx",
-    "check_training_mode",
-    "dequantize_helper",
-    "is_caffe2_aten_fallback",
-    "is_complex_value",
-    "parse_args",
-    "pytorch_name_to_type",
-    "quantize_helper",
-    "quantized_args",
-    "requantize_bias_helper",
-    "scalar_name_to_pytorch",
-    "scalar_type_to_onnx",
-    "scalar_type_to_pytorch_type",
-]
```

Comment:
I believe we need this as per PyTorch guidelines?

Reply:
Any link to the guildlines of this? I think __all__ is only needed if we need to do: import * from symbolic_helper and call, e.g. args_have_same_dtype without module. However, all the code calling helper methods in torch.onnx are scoped with symbolic_helper which is a recommended way in python.

Reply:
You are right, only now I noticed this is not a `__init__.py` file, so public APIs are not defined here in accordance with https://github.com/pytorch/pytorch/wiki/Public-API-definition-and-documentation?